### PR TITLE
Updated classes + ids in prWaitingRoom.html and its linked files

### PR DESCRIPTION
**Pull request by Mannendri**
I updated the classes and ids in the following files to match the Google Style Guide:

prWaitingRoom.html
firebase.js
prWaitingRoom.css

#56

### CODE REVIEW

File: public/components/firebase.js [hunk 1]
```diff
@@ -16,20 +16,20 @@ let roomDialog = null;
...
 function init() {
-  document.querySelector("#cameraBtn").addEventListener("click", openUserMedia);
-  document.querySelector("#hangupBtn").addEventListener("click", hangUp);
-  document.querySelector("#createBtn").addEventListener("click", createRoom);
-  document.querySelector("#matchMe").addEventListener("click", createAndJoin);
-  document.querySelector("#matchMe").disabled = false;
-  document.querySelector("#joinBtn").addEventListener("click", joinRoom);
-  document.querySelector("#joinBtn").disabled = true;
+  document.querySelector("#camera-btn").addEventListener("click", openUserMedia);
+  document.querySelector("#hang-up-btn").addEventListener("click", hangUp);
```
Comment (j0n3lson) [inline public/components/firebase.js:20]: One word? Maybe `hangup-up`?

File: public/templates/prWaitingRoom.html [hunk 2]
```diff
@@ -173,7 +173,7 @@ <h1 class="heading">
           hidden
           class="mdc-button mdc-button--raised"
           disabled
-          id="hangupBtn"
+          id="hang-up-btn"
```
Comment (j0n3lson) [inline public/templates/prWaitingRoom.html:176]: Ditto `hangup-btn`?

**Review by gideoncodenext — APPROVED** [review]
Good job with the changes! Everything looks good and makes sense to me.

**Comment by j0n3lson** [comment]
(pr-nagger-bot): Friendly ping. If you are a reviewer please take a look. If you are an author, please respond to comments.

File: public/assets/css/prWaitingRoom.css [hunk 3]
```diff
@@ -103,13 +103,13 @@ button {
...
+#local-video{
   box-shadow: 0 4px 10px 4px rgba(19, 35, 47, 0.3);
   margin-top: -50px;
 }
 
 
-#prMain{
+#pr-main{
   justify-self:center;
 /*   border: 1px black solid; */
```
Comment (j0n3lson) [inline public/assets/css/prWaitingRoom.css:114]: Dead code?

File: public/templates/prWaitingRoom.html [hunk 4]
```diff
@@ -104,36 +104,36 @@
...
 
     <h1 id="title">Read Patient Notes, then Join The Meeting!</h1>
-    <div id="pageDiv">
-      <div id="prMain">
+    <div id="page-div">
+      <div id="pr-main">
         <h1 class="heading">
           Join To Meet:
-          <p class="patientInfo" id="patientName">
+          <p class="patient-info" id="patient-name">
```
Comment (j0n3lson) [inline public/templates/prWaitingRoom.html:111]: Is there any good reason that the `class` doesn't match the `id`?

**Review by j0n3lson — CHANGES_REQUESTED** [review]
Check the `id` tags in these files as well:
- public/templates/reviewPage.html
- public/templates/firebase.html

File: public/templates/prWaitingRoom.html (hunk 4 above)
Comment (lillianchou94) [reply public/templates/prWaitingRoom.html:111]: I believe the "patient-info" class have css that applies to elements on line 118 and 123 as well and "patient-name" has its own specific css

**Review by lillianchou94 — APPROVED** [review]

File: public/components/firebase.js (hunk 1 above)
Comment (Mannendri) [reply public/components/firebase.js:20]: Yep, I'll change it to `hangup-btn`.

File: public/templates/prWaitingRoom.html (hunk 2 above)
Comment (Mannendri) [reply public/templates/prWaitingRoom.html:176]: Yes

File: public/assets/css/prWaitingRoom.css (hunk 3 above)
Comment (Mannendri) [reply public/assets/css/prWaitingRoom.css:114]: This is dead code, I'll delete it.

File: public/templates/prWaitingRoom.html (hunk 4 above)
Comment (Mannendri) [reply public/templates/prWaitingRoom.html:111]: This is true. The `patient-info`  class applies to three pieces of information that belong to the patient (i.e. their name, their issue, and the description of their issue). The `patient-name` id applies solely to the first piece of information - the patient's name. The two selectors have their own unique css rules.

**Review by j0n3lson — APPROVED** [review]

**Pull request merged**